arangod/RestHandler/RestCollectionHandler.cpp

```diff
@@ -435,33 +435,50 @@ void RestCollectionHandler::handleCommandPut() {
             opts.isSynchronousReplicationFrom =
                 _request->value("isSynchronousReplication");
 
-            auto trx = createTransaction(coll->name(), AccessMode::Type::EXCLUSIVE);
-            trx->addHint(transaction::Hints::Hint::INTERMEDIATE_COMMITS);
-            trx->addHint(transaction::Hints::Hint::ALLOW_RANGE_DELETE);
-            res = trx->begin();
+            _activeTrx = createTransaction(coll->name(), AccessMode::Type::WRITE);
```

Comment:
Why is the access level changed from exclusive to write here?
Exclusive was intentional, in order to avoid write-write conflicts and to speed up operations (no conflict checking etc.).